OHHTTPStubsDemo/Podfile

```diff
@@ -0,0 +1,5 @@
+source 'https://github.com/CocoaPods/Specs.git'
+xcodeproj 'OHHTTPStubsDemo.xcodeproj'
+platform :ios, '6.0'
+
+pod 'OHHTTPStubs', :git => 'https://github.com/AliSoftware/OHHTTPStubs.git', :branch => 'master'
```

Comment:
Why explicitly link on `master`?

Either don't constrain it on a version at all to get the latest, or convert it as a Development Pod (as I usually do for all my libs generally) by linking to a `:path`, but linking it to the remote and on master is not justified (and introduce bad practice for users not familiar with CP, as master is usually unstable anyway compared to an official release)